packages/build/bin/run-mocha.js

```diff
@@ -37,6 +37,12 @@ function run(argv, dryRun) {
     );
     mochaOpts.unshift('--require', sourceMapRegisterPath);
   }
+
+  // Fail any tests that are printing to console.
+  mochaOpts.unshift(
```

Comment:
I suggest that we add `--allow-console-logs` option (so that `@loopback/build` can be used by other modules outside of LoopBack 4).

1. `--allow-console-logs` is default to `false` if not present
2. `--allow-console-logs` can be used to override our check

Reply:
To be honest, I think this is [YAGNI](https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it). IMO, every developer out there should have a test suite that's not printing random stuff to console output. If somebody comes to us complaining about this particular feature of `lb-mocha`, only then it is the right time to implement the flag. It will also allow us to learn that somebody is actually using `@loopback/build`.

Reply:
Our CLI allows generated loopback apps to use `@loopback/build`. 

Since it's trivial to add this option to be consistent with other wrappers, I would moderately prefer to just add it now.

Reply:
Fair enough.